lib/artifact-search.js

```diff
@@ -68,29 +78,56 @@ module.exports = (bluebird, error, logger, requestOptions, requestPromise, respo
 
 
         /**
+         * @typedef {Object} searchParameters
+         * @property {(string|string[])} [search] A string or array of strings
+         *                                        for the search criteria. Can
+         *                                        be undefined (optional).
+         * @property {(string|string[])} [sort] A string or array of strings
```

Comment:
Sorry didn't catch this originally, I was told array of strings in a JSDoc block is `array.<string>`. Unless that changed, search and sort type should be updated. Did that change?